src/javascript/app_2/Assets/Common/icon-initial-logo-dark.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes  from 'prop-types';
+import React      from 'react';
+
+const IconInitialLogoDark = ({ className, classNamePath, classNameRect }) => (
```

Comment:
Btw does IconInitialLogoDark and IconInitialLogoLight need to be added to `icons.js`? @msamprz 

Also: className prop is not used + check the other travis errors 😅

Reply:
will do the fix 👍